src/DetectChanges/BCBreak/MethodBased/MethodParameterAdded.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\DetectChanges\BCBreak\MethodBased;
+
+use Psl\Str;
+use Roave\BackwardCompatibility\Change;
+use Roave\BackwardCompatibility\Changes;
+use Roave\BetterReflection\Reflection\ReflectionMethod;
+use Roave\BetterReflection\Reflection\ReflectionParameter;
+
+use function array_diff;
+use function array_map;
+
+/**
+ * Adding a parameter to a method on a non-final class is a BC break
```

Comment:
Let's add the "why" in this comment, and then we can merge :-)

Reply:
Nice! Updated :)